README.md

```diff
@@ -470,6 +471,7 @@ Table of Contents
   * [pagertree.com](https://pagertree.com/) - Simple interface for alerting and on-call management. Free up to 5 users.
   * [pingbreak.com](https://pingbreak.com/) — Modern uptime monitoring service. Check unlimited URLs and get downtime notifications via Discord, Slack or email.
   * [sematext.com](https://sematext.com/) — Free for 24 hours metrics, unlimited number of servers, 10 custom metrics, 500,000 custom metrics data points, unlimited dashboards, users, etc.
+  * [Scraper.AI)(https://scraper.ai) - SaaS that monitors website changes you're interested in and turns it into a consumable API for you to build on. Free 50 extractions and 10000 API calls / month.
```

Comment:
please list it in one catagory only

Reply:
I closed this PR in favour of https://github.com/ripienaar/free-for-dev/pull/1595